packages/ntcore-ts-client/src/lib/socket/socket.ts

```diff
@@ -124,12 +135,29 @@ export class NetworkTablesSocket {
         console.info('Robot Connected!');
         this.sendQueuedMessages();
 
+        if (this._websocket.protocol === 'v4.1.networktables.first.wpi.edu') {
+          // eslint-disable-next-line no-console
+          console.info('Connected on NT 4.1');
+          this.initializeRTTWebsocket();
+        } else {
+          // eslint-disable-next-line no-console
+          console.info('Connected on NT 4.0');
+        }
```

Comment:
Not sure if we need to broadcast this to users, it's very helpful for debugging though.